yb-voyager/cmd/exportDataStatusCommand.go

```diff
@@ -38,16 +44,36 @@ type exportTableMigStatusOutputRow struct {
 
 // Note that the `export data status` is running in a separate process. It won't have access to the in-memory state
 // held in the main `export data` process.
-func runExportDataStatusCmd() error {
-	exportSchemaDoneFlagFilePath := filepath.Join(exportDir, "metainfo/flags/exportSchemaDone")
-	_, err := os.Stat(exportSchemaDoneFlagFilePath)
+func runExportDataStatusCmdDbzm() error {
+	err := checkSchemaExportDone()
 	if err != nil {
-		if errors.Is(err, os.ErrNotExist) {
-			return fmt.Errorf("cannot run `export data status` before schema export is done")
+		return err
+	}
+	exportStatusFilePath := filepath.Join(exportDir, "data", "export_status.json");
+	status, err := dbzm.ReadExportStatus(exportStatusFilePath)
+	if err != nil {
+		utils.ErrExit("Failed to read export status file %s: %v", exportStatusFilePath, err)
+	}
+	tables := status.Tables
+	var exportStatusOutputRows []*exportTableMigStatusOutputRow
+	for _, table := range tables {
+		row := &exportTableMigStatusOutputRow{
```

Comment:
Can we include exported row count in the output (if we're using debezium)?

Reply:
Yes that can be done, but like with ora2pg we don't output that so uniformity won't be there. If that is not a problem, I can add it?